src/test/java/com/fasterxml/jackson/databind/deser/enums/EnumAltIdTest.java

```diff
@@ -213,4 +224,38 @@ public void testEnumWithAliasAndDefaultForUnknownValueEnabled() throws Exception
         MyEnum2352_3 multipleAliases2 = reader.readValue(q("multipleAliases2"));
         assertEquals(MyEnum2352_3.C, multipleAliases2);
     }
+
+    public void testEnumWithDefaultForUnknownValueEnabled() throws JsonProcessingException {
```

Comment:
Please just use `throws Exception` here since `JsonProcessingException` is renamed in 3.0 (master).